prow-configurations/config.yaml.tpl

```diff
@@ -38,6 +38,17 @@ presubmits: # runs on PRs
     spec:
       containers:
       - image: eu.gcr.io/kyma-project/snapshot/test/integration:0.0.1 # created by running `docker build -t <image> .` in the integration-job directory.
+  - name: kyma-gke-integration
```

Comment:
always update config.yaml.tpl and config.yaml <- please add that information to some Readme file